tololo/extensions/telegram_bot/lib/telegram/chains/done.ex

```diff
@@ -21,39 +21,55 @@ defmodule Tololo.Extensions.TelegramBot.Done do
   @impl true
   def match?(_, _), do: false
 
+  defp mark_delivery_done(delivery) do
+    Logger.info("Marking #{delivery.display_id} delivery as done")
+
+    %{state: new_state} =
+      delivery
+      |> Delivery.done_with_check!(nil, actor: @actor)
+
+    if new_state == "Delivery_Done" do
+      gettext("""
+      Delivery successfully marked as done.
+      """)
+    else
+      gettext("""
+      Error marking as done. You're not within the minimum range to mark delivery as done.
```

Comment:
The message should not be an Error. Since it must be reviewed by the admin before accepting it. So the message must be "Delivery marked with Problems. An admin will review it first".